packages/amplify-graphql-model-transformer/src/resources/dynamo-model-resource-generator.ts

```diff
@@ -25,28 +25,29 @@ export class DynamoModelResourceGenerator extends ModelResourceGenerator {
 
     if (this.isProvisioned()) {
       // add model related-parameters to the root stack
-      ctx.parameterManager.addParameter(ResourceConstants.PARAMETERS.DynamoDBModelTableReadIOPS, {
+      const rootStack = cdk.Stack.of(ctx.stackManager.scope);
+      new cdk.CfnParameter(rootStack, ResourceConstants.PARAMETERS.DynamoDBModelTableReadIOPS, {
```

Comment:
These are handled differently than the `env`/`apiName` etc. params since they are all defined with defaults, and while there could be work in the future to refactor sharing these a bit better (e.g. provide a hook to grab references to these in index/relational) this seems like a smaller potential change to CLI defined apps.

N.B. the CfnParameters MUST be created under the `cdk.Stack.of` the root scope, otherwise the resource logical id (which is used in all the refs) will break. The same will go for exports.